src/h5web/toolbar/LineToolbar.tsx

```diff
@@ -47,8 +47,6 @@ function LineToolbar(props: ToolbarProps): ReactElement {
         onScaleChange={setXScaleType}
       />
 
-      <Separator />
-
```

Comment:
I've removed the separator between the two selectors. Can't remember if we had talked about it when you first implemented it. I feel like it's more in line with what we've done so far of "grouping" controls of the same type (single toggle buttons, notably)

Reply:
Yes we did. I preferred having a clear separation between the two dropdowns. But in the end, it looks all right without it due to a bigger spacing between the two.